src/content/docs/setup/configuration/commerce-configuration.mdx

```diff
@@ -49,7 +49,7 @@ The `getHeaders` function is a helper function which takes a storefront scope (l
 
 ## Examples
 
-You can find the configuration file in your code repo at either `/config.json`, renamed from `/demo-config.json`. The file provides a template with the `keys` and `values` to configure the connection to the Commerce backend.
+For where the configuration file lives and how it's structured, see [Storefront configuration](/setup/configuration/commerce-configuration/). The examples below illustrate typical values only.
```

Comment:
Sorry - I realize now this is actually INSIDE the same file you are linking to, so it's a circular ref.

Let's just say - "here's some examples of typical values such as those generated by the config generator" (and in my PR actually you see I modified this line anyways).

I'm going to eventually delete `demo-config.json`...